projects/common-consumption/src/lib/content-details/toc-item/toc-item.component.html

```diff
@@ -4,7 +4,11 @@
       <sb-accordion-item [expanded]="isExpanded(index, item)" (collapsedChange)="collapsedChangeHandler($event);">
         {{ createUniqueId(tocData, item) }}
         <sb-accordion-header>
-          <span class="title">{{item?.name}}</span>
+            <div class="sb-checkbox sb-checkbox-primary" *ngIf="showCheckBox">
+                <input (change)="dispatchSelectedData(item)" [(ngModel)]="item.selected" type="checkbox" id="check1" name="example">
```

Comment:
@sri-pusuluri this should have only input & label. If showCheckbox is true, then only show checkbox else it will be hidden.
don't use div & span to hide & show.